pelican/generators.py

```diff
@@ -643,7 +643,7 @@ def generate_context(self):
         self.hidden_pages, self.hidden_translations = (
             process_translations(hidden_pages))
 
-        self._update_context(('pages', ))
+        self._update_context(('pages', 'hidden_pages'))
         self.context['PAGES'] = self.pages
```

Comment:
That sounds like a great thing to do in a separate PR.

On Thu, 18 Jun 2015 at 23:45 Deniz Turgut <notifications@github.com> wrote:

> In pelican/generators.py
> <https://github.com/getpelican/pelican/pull/1760#discussion_r32786859>:
>
> > @@ -643,7 +643,7 @@ def generate_context(self):
> >          self.hidden_pages, self.hidden_translations = (
> >              process_translations(hidden_pages))
> >
> > -        self._update_context(('pages', ))
> > +        self._update_context(('pages', 'hidden_pages'))
> >          self.context['PAGES'] = self.pages
>
> "issue a warning if it is called" will be tricky. Otherwise, I agree.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getpelican/pelican/pull/1760/files#r32786859>.
>